src/main/resources/templates/fragments/bootstrap_nav_header.html

```diff
@@ -92,13 +92,19 @@
                       </li>
                   </ul>
               </li>
-              <li th:unless="${isLoggedIn}">
+              <li th:unless="${isLoggedIn} and ${isGoogleOrGithub} == 0">
```

Comment:
this could just be isGithub == true if we define an isGithub method. same for the google form.